tee-worker/local-setup/launch.py

```diff
@@ -266,28 +275,38 @@ def main(processes, config_path, parachain_type, log_config_path, offset, parach
     parser = argparse.ArgumentParser(
         description="Run a setup consisting of a node and some workers"
     )
-    parser.add_argument("config", type=str, help="Config for the node and workers")
+    parser.add_argument("--config", type=str, help="Config for the node and workers")
     parser.add_argument(
-        "parachain",
+        "--parachain",
         nargs="?",
         default="local-docker",
         type=str,
         help="Config for parachain selection: local-docker / local-binary / remote",
     )
     parser.add_argument(
-        "log_config_path",
+        "--log_config_path",
```

Comment:
Can we have also a short flag for it - it's a bit long 😅 

And I believe conventionally the flag name is connected using `-` instead of `_`, like `--log-config-path`

Reply:
Updated ✅